mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -731,33 +731,38 @@ func expandAlertConfigurationNotification(d *schema.ResourceData) ([]matlas.Noti
 	return notifications, nil
 }
 
-func flattenAlertConfigurationNotifications(notifications []matlas.Notification) []map[string]interface{} {
+func flattenAlertConfigurationNotifications(d *schema.ResourceData, notifications []matlas.Notification) []map[string]interface{} {
+	notificationsSchema, err := expandAlertConfigurationNotification(d)
+	if err != nil {
+		return nil
+	}
```

Comment:
[q] Can you add a comment with the reason why we need this new schema? Thanks!

Reply:
@andreaangiolillo I pull in existing values set in TF file to set API keys in state file to ignore values that come back from API side that are masked